grpc/credentials_test.go

```diff
@@ -52,7 +50,7 @@ var _ = Describe("Test JSON and YAML to JSON credentials", func() {
 		func(filePath string) {
 			// Prepare test data
 			// #nosec: G304: Potential file inclusion via variable
-			data, err := ioutil.ReadFile(filePath)
+			data, err := os.ReadFile(filePath)
```

Comment:
Why this change?

Reply:
`ioutil` is deprecated. Linter was complaining about it